test/test_formats.py

```diff
@@ -74,8 +79,12 @@ def setUp(self):
         shutil.copy(self.original, self.filename)
         config.setting = settings.copy()
         QtCore.QObject.tagger = FakeTagger()
+        self.log_messages = []
+        log.main_logger.unregister_receiver(log._stderr_receiver)
```

Comment:
To avoid e.g. warning messages appearing in console during test. But I don't see why that is necessary. Will delete.